exporters/otlp/otlptrace/README.md

```diff
@@ -0,0 +1,26 @@
+# OpenTelemetry-Go OTLP Span Exporter
+
+OpenTelemetry Protocol (OTLP) Span Exporter.
+
+To constructs a new OTLP trace Exporter, you can follow this 
+ [`example`](https://github.com/open-telemetry/opentelemetry-go/blob/main/exporters/otlp/otlptrace/example_test.go).
+
+## Installation
+
+```
+go get -u go.opentelemetry.io/otel/exporters/otlp/otlptrace
+```
+
+## [`otlptrace`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace)
+
+The `otlptrace` package provides an exporter implementing the OTel span exporter interface.
+This exporter is configured using a client satisfying the `otlptrace.Client` interface.
+This client handles the transformation of data into wire format and the transmission of that data to the collector.
+
+## [`otlptracegrpc`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc)
+
+The `otlptracegrpc` package implements a gRPC client to be used in the span exporter.
+
+##  [`otlptracehttp`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp)
+
+The `otlptracehttp` package implements a HTTP client to be used in the span exporter.
```

Comment:
We miss information about configuration via environmental variables. I am not sure which are supported (you would need to check the code/godoc/changelog). Here are the docs for the env vars that in theory should be supported: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md.

Here is a README how the env vars are documented for Jaeger exporter: https://github.com/open-telemetry/opentelemetry-go/blob/main/exporters/jaeger/README.md#environment-variables

Reply:
I hope this will help you: https://github.com/open-telemetry/opentelemetry-go/blob/d5d4c87f6d2581fdcd3f716155912f7be66dfd85/exporters/otlp/otlpmetric/internal/otlpconfig/envconfig.go#L70-L138

Reply:
I hope this will help you; https://github.com/open-telemetry/opentelemetry-go/blob/d5d4c87f6d2581fdcd3f716155912f7be66dfd85/exporters/otlp/otlptrace/internal/otlpconfig/envconfig.go#L70-L137

Reply:
@pellared can you add this in a follow on PR?

Reply:
@MrAlias Added to my TODO list. I will try to do it next week.

Reply:
> @MrAlias Added to my TODO list. I will try to do it next week.

:tada: thanks :smile:

Reply:
PR: https://github.com/open-telemetry/opentelemetry-go/pull/2222